pkg/config/config.go

```diff
@@ -2150,6 +2159,13 @@ func NormalizeConfig(cfg *Config, cfgMetadata config_loader.YAMLMetadata) (err e
 	nlog.WithField("DnsHostnameResolution", cfg.DnsHostnameResolution).Debug("DNS hostname resolution.")
 	// IgnoreReclaimable value defined in NewConfig
 	nlog.WithField("IgnoreReclaimable", cfg.IgnoreReclaimable).Debug("Ignoring reclaimable memory.")
+
+	// CloudType default value if cloud is unknown
+	if !cloud.Type(cfg.CloudProvider).IsValidCloud() {
```

Comment:
I only would print a warning if the config option is filled and not valid. If it's empty (default) I would not print it as it can be confusing.